portfolio/src/main/webapp/script.js

```diff
@@ -398,3 +399,24 @@ function buildInfoWindowInput(lat, lng) {
 
   return containerDiv;
 }
+
+function drawChart() {
+  fetch('/ufos-data').then(response => response.json())
+    .then((ufosSightings) => {
+      const data = new google.visualization.DataTable();
+      data.addColumn('string', 'Year');
+      data.addColumn('number', 'Sightings');
+      Object.keys(ufosSightings).forEach((currentYear) => {
+      data.addRow([currentYear, ufosSightings[currentYear]]);
+  });
+
+  const options = {
+    'title': 'Ufos Sightings',
+    'width':600,
```

Comment:
This won't play well on the phone. If you have time, check out about "responsive web design" on techniques how to overcome these problems. It is a big subject.